moped-database/migrations/1649349626445_alter_project_length_field_drop_compute_project_length_function/down.sql

```diff
@@ -0,0 +1,21 @@
+-- Add start_date column back
+ALTER TABLE "public"."moped_project" ADD COLUMN "start_date" date;
+
+-- Add compute_project_length function
+-- Name: compute_project_length(); Type: FUNCTION; Schema: public; Owner: -
+
+CREATE FUNCTION public.compute_project_length() RETURNS integer
+    LANGUAGE sql STABLE
+    AS $$
+SELECT
+case when moped_project.end_date is null
+then date(now()) - moped_project.start_date
+else moped_project.end_date - moped_project.start_date
+end as project_length
+FROM moped_project
+$$;
+
+-- Drop null default and set default to compute_project_length
+ALTER TABLE "public"."moped_project"
+    ALTER COLUMN "project_length" DROP DEFAULT,
+    ALTER COLUMN "project_length" SET DEFAULT public.compute_project_length();
```

Comment:
This down.sql file does not seem to restore the database to the state that it would have been in prior to running the associated up.sql commands. Also, I'm not sure these changes fully capture the full task, as I understand it. If you have clearer direction than what I'm aware of, please forgive this feedback:

The up.sql does:
* Drop the default value of the `project_length` field 🆗  
* Set the default value to `null` for the same field 🆗 
* Drop the function `compute_project_length ✅ 
* It does not drop the `end_date` column ❌ 
* It does not drop the `project_length` column ❌ 

Then the down.sql:
* Adds back the `start_date` column ❓ 
  * This is not a restoration of something happened in the up.sql. If you needed to do schema changes, they should come in matching pairs and in separatete migrations.
* recreates the function, which is good and should be in there. ✅ 
* Restores the function as the default value for `project_length` ✅

Reply:
I agree with Frank that this file needs to be updated to not add back the start_date, since that wasn't addressed in its corresponding up.sql file. 

Personally I think the end_date and project_length columns aren't needed, but I do remember that your slack comment about this issue stated you weren't going to address that in this PR, but rather we would discuss as a team in the future.

Reply:
I gave Tilly the direction to leave `end_date` and `project_length` in the database for now. This was after asking @amenity if they were needed. She said yes, that other divisions may require those fields when we import their project data. I proposed we make a backlog issue to address project `end_date` and `project_length` later.